tests/schema/__snapshots__/app.test.ts.snap

```diff
@@ -1438,6 +1438,7 @@ Object {
             "name": "id",
           },
         ],
+        "isOneOf": true,
```

Comment:
@Weakky maybe we want to not snapshot this since it is going to break every time.

And when it does change, I don't really make sense of it. For example now.